5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -5,8 +5,14 @@
 # volume to override it.
 export MYSQL_DATADIR=/var/lib/mysql/data
 
+# Get prefix rather than hard-code it
+export MYSQL_PREFIX=$(which mysqld_safe|sed -e 's|/bin/mysqld_safe$||')
+
+# Path to MySQL scripts
+export MYSQL_SCRIPTS_PATH=/usr/share/container-scripts/mysql
```

Comment:
as discussed above, defining this in the dockerfile seems reasonable.  perhaps give the variable a more generic name to make it easier to copy/paste it between dockerfiles for other products?